server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -205,17 +212,18 @@ private DivTag createImageUploadForm(ProgramDefinition program) {
             // for more context.
             .with(fileInputElement);
 
-    // TODO(#5676): Replace with final UX once we have it.
-    return div()
-        .with(fullForm)
-        .with(
-            submitButton("Save image")
-                .withForm(IMAGE_FILE_UPLOAD_FORM_ID)
-                .withClasses(ButtonStyles.SOLID_BLUE, "mb-2"))
-        .with(fileUploadViewStrategy.footerTags());
+    DivTag imageUploadForm =
+        div()
+            .with(fullForm)
+            .with(
+                submitButton("Save image")
+                    .withForm(IMAGE_FILE_UPLOAD_FORM_ID)
+                    .withClasses(ButtonStyles.SOLID_BLUE, "mb-2"))
+            .with(fileUploadViewStrategy.footerTags());
+
+    return div().with(imageUploadForm).with(deleteImageButton);
```

Comment:
suggestion (non-blocking): keep this method the same and create a div that you add the delete button to on line 125 eg `.with(div().with(createImageUploadForm(programDefinition).with(deleteImageModal.getButton()))`

that way you don't have to pass through `deleteImageButton` just to add it onto the div.

Reply:
Oh that's a great idea! Done.